apps/io-wallet-user-func/local.settings.json.example

```diff
@@ -29,6 +29,8 @@
     "EntityConfigurationStorageAccount__serviceUri": "ENTITY_CONFIGURATION_STORAGE_ACCOUNT__SERVICE_URI",
     "EntityConfigurationStorageContainerName": "ENTITY_CONFIGURATION_STORAGE_CONTAINER_NAME",
     "StorageConnectionString": "STORAGE_ACCOUNT_CONNECTION_STRING",
+    "WalletInstanceCreationQueueName": "WALLET_INSTANCE_CREATION_QUEUE_NAME",
```

Comment:
Don't we want to give the queue a name that refers to the fact that it is sending the email rather than referring to the creation of the wallet instance?